app/Http/Controllers/Auth/GetAuthUserController.php

```diff
@@ -19,7 +19,9 @@ public function __invoke(): JsonResponse
         /** @var User $user */
         $user = auth()->user();
 
-        $user->load('categories');
+        if ($user->type !== UserTypesConstant::ADMIN) {
+            $user->load('categories');
+        }
```

Comment:
The user model has a utility function to verify if the record is an `ADMIN`. Use that function.